baybe/recommenders/pure/bayesian/botorch.py

```diff
@@ -169,7 +169,7 @@ def _recommend_continuous(
             A dataframe containing the recommendations as individual rows.
         """
         # For batch size > 1, this optimizer needs a MC acquisition function
```

Comment:
With the new explicit flag, this comment now becomes very much redundant – which indeed confirms that the underlying logic is now clearer 👍🏼 
```suggestion
```